modules/virt-runbook-vmstorageclasswarning.adoc

```diff
@@ -54,29 +54,10 @@ $ oc get pv <pv_name> -o=jsonpath='{.spec.storageClassName}'
 ----
 
 [discrete]
-[id="mitigation-virtualmachinecrcerrors"]
+[id="mitigation-vmstorageclasswarning"]
 == Mitigation
 
-Add the `krbd:rxbounce` map option to the storage class configuration to use
-a bounce buffer when receiving data:
-
-[source,yaml]
-----
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: vm-sc
-parameters:
-  # ...
-  mounter: rbd
-  mapOptions: "krbd:rxbounce"
-provisioner: openshift-storage.rbd.csi.ceph.com
-# ...
-----
-
-The `krbd:rxbounce` option creates a bounce buffer to receive data. The default
-behavior is for the destination buffer to receive data directly. A bounce buffer
-is required if the stability of the destination buffer cannot be guaranteed.
+Create a default OpenShift Virtualization storage class with the"krbd:rxbounce" map option. See  link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
```

Comment:
@sbeskin-redhat The link to "Optimizing ODF PersistentVolumes for Windows VMs" is referenced here twice..

Reply:
@sradco  Fixed, thank you